### PR TITLE
add history.peek() and 'exitmode' event

### PR DESCRIPTION
**Pull request by mvayngrib**

### CODE REVIEW

File: dist/vorpal.js [hunk 1]
```diff
@@ -990,6 +990,7 @@ vorpal._exitMode = function (options) {
   ssn._mode = false;
   this.cmdHistory.exitMode();
   ssn.modeDelimiter(false);
+  this.emit('exitmode', this.cmdHistory.peek());
```
Comment (dthree) [inline dist/vorpal.js]: Can we name this `mode_exit` to fit more in line with the other event semantics?

https://github.com/dthree/vorpal/wiki/Docs-|-Events

Thanks.

Comment (mvayngrib) [inline dist/vorpal.js]: sure, will update

Comment (dthree) [inline dist/vorpal.js]: :+1:

**Comment by dthree** [comment]
Thanks, awesome! :+1:

**Comment by mvayngrib** [comment]
@dthree yay, can you release an npm version?

**Pull request merged**